src/c++/perf_analyzer/client_backend/triton_c_api/triton_loader.cc

```diff
@@ -945,7 +945,8 @@ TritonLoader::Infer(
       TRITONSERVER_MemoryType memory_type;
       int64_t memory_type_id;
       RETURN_IF_ERROR(shm_manager_->GetMemoryInfo(
-          shm_name, offset, &buf, &memory_type, &memory_type_id));
+          shm_name, offset, shm_byte_size, &buf, &memory_type,
```

Comment:
looks like this is missing the changes in the `shared_memory_manager.cc` file.

Reply:
Currently it looks like it's matching for `GetMemoryInfo`. What is the problem you are referring to?